### PR TITLE
Addition of requirements.txt and edits to readMe.md

### PR DESCRIPTION
**Pull request by georgebangeboss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+Scrapes the web for recently posted mechanical engineering jobs in Kenya
+## Installation Guide
 
 1. Make sure python 3.9 or higher is installed in your machine
-2. Pip install required modules (BeautifulSoup, lxml, requests)
-3. Internet connection required to run python file
-4. Run the command below in your terminal
-5. python main.py
\ No newline at end of file
+2. Create virtual environment and activate it
```
Comment (byronkenoly) [inline README.md]: Kindly add the command for creating and activating the virtual environment

**Review by byronkenoly — CHANGES_REQUESTED** [review]
Update README.md
Add command for creating and activating virtual environment

File: README.md (hunk 1 above)
Comment (georgebangeboss) [reply README.md]: on it

**Review by byronkenoly — APPROVED** [review]
New file added: requirements.txt - contains dependencies required to run the program
Modified: README.md - beefed up documentation

Thanks for your contributions Mr. Bange

**Pull request merged**